packages/entities/entities-routes/src/components/RouteFormExpressionsEditor.vue

```diff
@@ -4,21 +4,108 @@
     v-model="expression"
     :schema="schema"
   />
+
+  <KTooltip
+    placement="bottom-start"
+    :text="tooltipText"
+  >
+    <div
+      v-if="showExpressionsModalEntry"
+      :class="{ disabled: !isHttpBasedProtocol, 'route-form-open-in-playground': true }"
+      data-testid="open-router-playground"
+      @click="handleOpenPlayground"
+    >
+      <RocketIcon size="16px" />
```

Comment:
```suggestion
      <RocketIcon size="KUI_ICON_SIZE_30" />
```

suggestion: use from `@kong/design-tokens`